typings/index.d.ts

```diff
@@ -21,21 +21,41 @@ declare enum ChannelTypes {
   STAGE = 13,
 }
 
-declare enum StickerFormatTypes {
-  PNG = 1,
-  APNG = 2,
-  LOTTIE = 3,
+declare enum InteractionResponseTypes {
+  PONG = 1,
+  CHANNEL_MESSAGE_WITH_SOURCE = 4,
+  DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE = 5,
+}
+
+declare enum InteractionTypes {
+  PING = 1,
+  APPLICATION_COMMAND = 2,
 }
 
 declare enum OverwriteTypes {
   role = 0,
   member = 1,
 }
 
+declare enum StickerFormatTypes {
+  PNG = 1,
+  APNG = 2,
+  LOTTIE = 3,
+}
```

Comment:
Import from dapi-typessss

Reply:
Wouldn't that be out of scope? There's probably a lot more that can be imported

Reply:
InteractionResponseTypes and InteractionTypes tho?

Reply:
Those are pascal case in discord-api-types instead of fully upper case, so not the same thing